src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -0,0 +1,306 @@
+
+(2) Definition of pickup_type and drop_off_type:
+
+In the `ServiceJourneyPattern/pointsInSequence/StopPointInJourneyPattern` corresponding to this `stop_time` (see `(1)`):
+* if the `ForBoarding` node is existing and with a `False` value, `pickup_type` is set to "1" (no boarding)
+* else `pickup_type` is set to "0" (regular boarding)
+`drop_off_type` is set using the same method and using the `ForAlighting` node.
+
+(3) Definition of local_zone_id:
+
+The declaration of those zones is made in the nodes `RoutingConstraintZone` of the **offre_** file.
+The `local_zone_id` is specified with an auto-incremented integer. Each `RoutingConstraintZone/@id` is associated with a new integer. 
+This `RoutingConstraintZone` contains a list of `ScheduledStopPointRef` (in `RoutingConstraintZone/members/ScheduledStopPointRef/@ref`). If a stop_time corresponds to one of those `ScheduledStopPointRef`, the `local_zone_id` is set to the associated integer.
+
+### calendar.txt and calendar_dates.txt
+Active days of `trips` are decribed in **calendriers.xml** of each folder in the `GeneralFrame/members/DayType` node (with the use of `DayTypeAssignment` and `OperatingPeriod`).
+
+A `trip` references one or several `DayType` in `ServiceJourney/dayTypes/DayTypeRef/@ref`.
+The `service_id` property of the calendar in the NTFS is specified by an auto-incremented integer (this field is prefixed).
+
+Active dates of the calendar are specified by:
+- `DayType` nodes describing the active days of a week
+  + days of the week are listed in `DayType/properties/PropertyOfDay/DaysOfWeek nodes`. 
+  + Expected values MUST be one of `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday`, `Saturday`, `Sunday`.
+- `OperatingPeriod` nodes describing periods (basically a begining date and an end date) referenced in `DayTypeAssignment`
+- `DayTypeAssignment` nodes with 2 possible uses:
+  + Activate or deactivate a specific day on a DayType (with the nodes `IsAvailable` and `Date`)
+  + Apply the active days of the referenced DayType on an `OperatingPeriod`
+
+All resulting calendars are to be restricted between `ValidBetween/FromDate` and `ValidBetween/ToDate` specified at the top level of the file in `GeneralFrame`.
+
+
+Be careful: Definition of calendars and exceptions in calendar_dates may not be the same definition as the one in the Netex-IDFM files, but the resulting active dates will be the same.
+
+Here is 3 possible modelizations in Netex-IDFM of a calendar running from 2016-07-01 to 2016-07-31 from monday to saturday except on 2016-07-14:
+![](./netex_idfm_to_ntfs_calendars.png "Definition of calendars in Netex-IDFM specs")
+
+
+### comments.txt
+`comments` are provided in **commun.xml** file in the nodes **GeneralFrame/members/Notice**.
+
+NTFS field | Netex IDFM element | Mapping rule/Comment
+--- | --- | ---
+comment_id | *Notice/@id* | This field is prefixed.
+comment_name | *Notice/Text* |
```

Comment:
There is no comment type? :(

Reply:
There is a `TypeOfNoticeRef`, but in this profile the possible values are `ServiceJourneyNotice` or `LineNotice` only...